Danielshow/naijahacks-achievers
# Ch set up project

## What does this PR do?
This PR set up event rock project with various tools needed

## Description of Task to be completed
- Add babel to transpile code from es6 to es5
- Add eslint to follow AirBnb style guide
- Add postgresql to make data persist
- Add express to create server
- Add dotenv module to use environment variable

## How should this be manually tested
```
clone this branch
npm install to install all dependencies
create .env file containing parameters of your postgres database
check package.json
```
#### Screenshot
